docker-compose.yml.example

```diff
@@ -0,0 +1,15 @@
+version: '3'
```

Comment:
I got an error with version 3 (see below). Changing it to version 2 resolved it.
ERROR: Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a version of "2" (or "2.0") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.